source/jormungandr/jormungandr/scenarios/helper_classes/streetnetwork_path.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) 2001-2017, Canal TP and/or its affiliates. All rights reserved.
+#
+# This file is part of Navitia,
+#     the software to build cool stuff with public transport.
+#
+# Hope you'll enjoy and contribute to this project,
+#     powered by Canal TP (www.canaltp.fr).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# IRC #navitia on freenode
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+import helper_future
+from jormungandr import utils
+from jormungandr.street_network.street_network import StreetNetworkPathType
+import logging
+
+class StreetNetworkPath:
+    """
+    A StreetNetworkPath is a journey from orig_obj to dest_obj purely in street network(without any pt)
+    """
+    def __init__(self, instance, orig_obj, dest_obj, mode, fallback_extremity, request, streetnetwork_path_type):
+        """
+        :param instance: street network connector, may be kraken, valhalla, etc.
```

Comment:
may be worth a rename : `sn_instance` (to avoid mixing with navitia's coverage instance elsewhere